edx_notifications/server/web/templates/django/notifications_widget.html

```diff
@@ -0,0 +1,31 @@
+{% comment %}
+
+This is a "drop-in" Django template widget to instantiate the edx-notifications front end application
+which is a Backbone application that is defined in edx_notifications/server/web/static/js
+
+This HTML snippet assumes that jQuery been included in the page that is using this widget
+
+{% endcomment %}
+
+<div id='edx_notification_container'></div>
+<div id="edx_notification_pane"></div>
+
+<script src="{{ STATIC_URL }}edx_notifications/js/vendor/underscore.js"></script>
+<script src="{{ STATIC_URL }}edx_notifications/js/vendor/backbone.js"></script>
+
+<script src="{{ STATIC_URL }}edx_notifications/js/models/counter_icon_model.js"></script>
+<script src="{{ STATIC_URL }}edx_notifications/js/models/user_notification_model.js"></script>
+<script src="{{ STATIC_URL }}edx_notifications/js/models/notification_collection_model.js"></script>
+
+<script src="{{ STATIC_URL }}edx_notifications/js/views/counter_icon_view.js"></script>
+<script src="{{ STATIC_URL }}edx_notifications/js/views/notification_pane_view.js"></script>
+
+
+<script type="text/javascript">
+    /* start up the root Notification view, which will bootstrap the whole front-end */
+    var counter_icon = new CounterIconView({
```

Comment:
In order to use RequireJS Optimizer, we are using a pattern now where Mako templates include a single JS file (which we typically call a factory) which knows how to construct the models, views etc needed. Even if you don't use RequireJS yet, it would probably be good to follow this pattern so that we can adopt it in the future.

See this wiki page for more details:

https://openedx.atlassian.net/wiki/display/TNL/How+to+add+a+new+page+to+Studio